src/ducks/SANCharts/ChartPage.js

```diff
@@ -361,6 +379,23 @@ class ChartPage extends Component {
                     />
                   </div>
                 )}
+
+                {!hideSettings.linkToDashboard && (
+                  <div className={styles.moreData}>
+                    <UpgradeBtn className={styles.upgradeBtn}>
+                      Get more data
+                    </UpgradeBtn>
+
+                    <div className={styles.limited}>
+                      <span className={styles.limitedLabel}>
+                        <Link to='/dashboards'>Limited data</Link>
+                      </span>
+                      <HelpPopup position='bottom left'>
+                        See much more data in our SANbase Dashboards
+                      </HelpPopup>
+                    </div>
+                  </div>
+                )}
```

Comment:
1. These layout is outdated;
2. PR should contain changes of one subject/task, otherwise they bloat in size. Moving working implementation of the `ChartWidget` is enough.